frontend/src/animal/AnimalListItem.tsx

```diff
@@ -1,15 +1,22 @@
 import React from "react";
 import {Animal} from "./Animal";
+import {useNavigate} from "react-router-dom";
 
 export default function AnimalListItem(props:
                                            {
                                                animal: Animal,
                                                onDeleteAnimal: (id: string) => Promise<void>,
                                            }) {
 
+    const navigate = useNavigate();
+
     return (
         <li key={props.animal.id}>{props.animal.name}
             <button onClick={() => props.onDeleteAnimal(props.animal.id)}>Delete</button>
+            <button onClick={() => {
+                navigate(`/${props.animal.id}`)
```

Comment:
in der Ticketbeschreibung wird ein anderer Aufbau der URL gefordert: /#/animals/animalId